### PR TITLE
Multi architechture and Build without kraken

### PR DESCRIPTION
**Pull request by sahilph**
## Overview
I was able to build the docker image for v2.1.1 without using kraken. The previous versions (v2.1.0 and v2.0.0) cannot be compiled as of now. (Check #19). I have removed their building from travis.
Also I have added v1.7.1 as per @dogi's request.

Multiarch should work once the branch has been merged with master.

## Testing recommendations
Use the following images:
For v2.1.1: treehouses/rpi-couchdb:2.1.1-0.0.1-MA_and_build-without-kraken-712df3cb
For v1.7.1: treehouses/rpi-couchdb:1.7.1-0.0.1-MA_and_build-without-kraken-712df3cb

## Checklist

- [x] Code is written and works correctly;
- [x] Documentation reflects the changes;

### CODE REVIEW

File: .travis/deploy_docker_rpi.sh [hunk 1]
```diff
@@ -61,30 +55,18 @@ random_generator(){
...
-deploy_v210
-build_message Buil V210 image finished, check build result!
+build_message Build V171 image started...
+deploy_v171
+build_message Build V171 image finished, check build result!
 
 build_message Build V211 image started...
 deploy_v211
-build_message Buil V211 image finished, check build result!
+build_message Build V211 image finished, check build result!
```
Comment (mappuji) [inline .travis/deploy_docker_rpi.sh]: Thank you for this

File: .travis/travis_utils.sh [hunk 2]
```diff
@@ -29,14 +29,20 @@ prepare_package(){
 	V210_DOCKER_NAME_LATEST=$DOCKER_ORG/$DOCKER_REPO:2.1.0
 	V211_DOCKER_NAME=$DOCKER_ORG/$DOCKER_REPO:2.1.1-$VERSION-$BRANCH-$COMMIT
 	V211_DOCKER_NAME_LATEST=$DOCKER_ORG/$DOCKER_REPO:2.1.1
+    V171_DOCKER_NAME=$DOCKER_ORG/$DOCKER_REPO:1.7.1-$VERSION-$BRANCH-$COMMIT
```
Comment (mappuji) [inline .travis/travis_utils.sh]: [small] homogeneous indentation is always a good idea.

**Comment by mappuji** [comment]
Thanks for the style fix @sahilph

**Comment by sahilph** [comment]
no problem @mappuji 😃

**Pull request merged**